docs/admin/upgrading.rst

```diff
@@ -23,60 +24,51 @@ Advice
 Basic instructions
 ==================
 
-Composer Upgrade
-----------------
-
-If you had your Elgg 2.3 project installed using composer, you can follow this sequence:
-
-.. code-block:: sh
-
-    composer self-update
-
-    cd ./path/to/project/root
-    composer require elgg/elgg:~3.0
-    composer update
-    vendor/bin/elgg-cli upgrade async -v
-
-
-Manual Upgrade
---------------
-
-#. Log in as an admin to your site
-#. Disable caching in Advanced Settings
-#. **Back up your database, data directory, and code**
-#. Download the new version of Elgg from http://elgg.org
-#. Update the files
-    * If doing a patch upgrade (1.9.x), overwrite your existing files with the new version of Elgg
-    * If doing a minor upgrade (1.x), replace the existing core files completely
-#. Merge any new changes to the rewrite rules
-    * For Apache from ``install/config/htaccess.dist`` into ``.htaccess``
-    * For Nginx from ``install/config/nginx.dist`` into your server configuration (usually inside ``/etc/nginx/sites-enabled``)
-#. Merge any new changes from ``settings.example.php`` into ``settings.php``
-#. Visit http://your-elgg-site.com/upgrade.php
-
-.. note::
-
-   Any modifications should have been written within plugins, so that they are not lost on overwriting.
-   If this is not the case, take care to maintain your modifications. 
-
 From 2.3 to 3.0
 ===============
 
-Update ``settings.php``
+1. Update ``composer.json``
+---------------------------
+
+If you have used Elgg's starter project to install Elgg 2.3, you may need to update your ``composer.json``:
+
+ * change platform requirements to PHP >= 7.0
+ * optional set autoloader optimization parameters
+ * optionally disable fxp-asset plugin in favor of asset-packagist
+
+Your ``composer.json`` would look something like this (depending what changes you may have introduced yourself):
+
+.. code-block:: json
+
+   {
+      "type": "project",
+	   "name": "elgg/starter-project",
+	   "require": {
+		   "elgg/elgg": "3.*"
+	   },
+      "config": {
```

Comment:
align

Reply:
I wasn't going to say anything ;)

Reply:
It's actually messed up in the starter project